src/Autofac.Extras.MvvmCross/AutofacMvxIocProvider.cs

```diff
@@ -634,19 +683,43 @@ public virtual bool TryResolve(Type type, out object resolved)
                 throw new ArgumentNullException(nameof(registration));
             }
 
-            if (this.PropertyInjectorOptions.InjectIntoProperties == MvxPropertyInjection.None)
+            var options = this.PropertyInjectionOptions as IAutofacPropertyInjectorOptions;
```

Comment:
Looks like the `IPropertySelector` may have been better passed as a separate parameter in the previous implementation. I'm not sure how they are supposed to fit into the DI abstraction as the `IMvxIoCProvider` doesn't reference them directly.